core/src/main/scala/filodb.core/query/QueryConfig.scala

```diff
@@ -51,7 +51,11 @@ object QueryConfig {
                                            remoteGrpcEndpoint = None,
                                            enforceResultByteLimit = false,
                                            allowPartialResultsRangeQuery = false,
-                                           allowPartialResultsMetadataQuery = true)
+                                           allowPartialResultsMetadataQuery = true,
+                                           recordContainerOverrides =
+                                             Map("filodb-query-exec-localpartitionreduceaggregateexec-topbottomk"
+                                                     -> 40960,
+                                                  "filodb-query-exec-metadataexec" -> 8192))
```

Comment:
Shouldn't you be using the `container-size-overrides` config ?

Reply:
Actually this is used for a unit where where construction should fail and when using 64 it doesn't.